server/pxf-hive/src/main/java/org/greenplum/pxf/plugins/hive/HiveResolver.java

```diff
@@ -150,7 +150,7 @@ public List<OneField> getFields(OneRow onerow) throws Exception {
      */
     @Override
     public OneRow setFields(List<OneField> record) {
-        throw new UnsupportedOperationException();
+        throw new UnsupportedOperationException("Current operation is not supported");
```

Comment:
The method `setFields` is called in the write path so this should be
```suggestion
        throw new UnsupportedOperationException("Hive resolver does not support write operation.");
```